validator/sawtooth_validator/execution/processor_handlers.py

```diff
@@ -33,7 +33,46 @@
 MAX_SDK_PROTOCOL_VERSION = 1
 
 
+class ProcessorRegisterValidationHandler(Handler):
+    """Validates the TP Register Request and emits an Ack"""
+    def handle(self, connection_id, message_content):
+        request = processor_pb2.TpRegisterRequest()
+        request.ParseFromString(message_content)
+
+        # Reject the request if requested version cannot be handled,
+        # validator does backward compatible support.
+        if request.protocol_version > MAX_SDK_PROTOCOL_VERSION:
+            ack = processor_pb2.TpRegisterResponse()
+            ack.status = ack.ERROR
+            # Send protocol_version of validator, so that the SDK can cross
```

Comment:
I realize this is moved code, but does this comment make sense in the context of an error response? It might clean it up a bit to move the ack and protocol_version lines in both sections above the if

Reply:
That looks like it was some copy-paste of the same message.  Likewise, I wonder if the validator should send back the version it supports in the error condition.

Reply:
The latter probably should be in a separate PR, though.